src/main/java/lv/ctco/javaschool/vote/boundary/VoteApi.java

```diff
@@ -51,6 +52,18 @@ public EventDtoList getIncompleteEvents() {
         return evList;
     }
 
+    @GET
+    @RolesAllowed({"ADMIN", "USER"})
+    @Path("/checkDay")
+    public boolean getDay() {
```

Comment:
Looks like the method should be renamed `checkDay()` to match its purpose.

Reply:
Method renamed